FinSearchUnified/Bundle/SearchBundleFindologic/SortingHandlerInterface.php

```diff
@@ -20,12 +21,12 @@ public function supportsSorting(SortingInterface $sorting);
      * Handles the passed sorting object.
      *
      * @param SortingInterface $sorting
-     * @param QueryBuilder $query
+     * @param NewQueryBuilder $query
      * @param ShopContextInterface $context
      */
     public function generateSorting(
         SortingInterface $sorting,
-        QueryBuilder $query,
+        NewQueryBuilder $query,
```

Comment:
I'd suggest using the interface here.

Reply:
We do not have any interface for this. We have an abstract class, which is being used here. (Applicable to other comments below as well)